contributor-guide/modules/ROOT/pages/docs/linking-guide.adoc

```diff
@@ -0,0 +1,245 @@
+= Linking Guide
+:navtitle: Linking Guide
+
+This section provides best practices on how to link from your library documentation, to other components of Boost. This includes the full set of libraries, any individual library, and the *master* and *develop* branch of both the super-project and any individual library.
+
+All the links or link macros provided in this section are the recommended ones to copy and paste into your topics.
+
+== Links to the Super-Project Source
+
+The Boost super-project contains all library source files, documentation, tools and build process. The *Master* branch contains the release candidate.
+
+=== Master Branch
+
+The *master* version of the super-project:
+
+* https://github.com/boostorg/boost/tree/master
+
+And to see all the library sources available in the super-project:
+
+* https://github.com/boostorg/boost/tree/master/libs
+
+Note:: If you now click on one of the libraries, notice the URL changes to include a GUID.
+
+=== Develop Branch
+
+Similarly to the *master* branch, the *develop* branch follows the same format:
+
+* https://github.com/boostorg/boost/tree/develop
+
+* https://github.com/boostorg/boost/tree/develop/libs
+
+
+== Individual Library Sources
+
+If you click on an individual library using the links above, you will be able to access the source code for that library, usually in an `include` sub-folder. There may also be an `example` or `examples` folder, that you may well want to link to from your library documentation. 
+
+For example, the following link accesses the boost:beast[] examples: https://github.com/boostorg/beast/tree/c316c6bd3571991aeac65f0fc35fca9067bc7906/example
+
+It is OK to include these links, with the GUIDs, in your documentation. Make sure to check that you retrieve the link for the *master* branch.
+
+== All Libraries Documentation
+
+To link to a single page containing lists of all libraries, where each entry is a link to that library's documentation, use the following format:
+
+[source,acsiidoc]
+----
+https://www.boost.org/doc/libs/<VERSION>/libs/libraries.htm
+----
+
+For example, for the latest current version:  https://www.boost.org/doc/libs/1_83_0/libs/libraries.htm
+
+== Individual Library Links
+
+If you are writing library documentation using the AsciiDoc/Antora system, you can use a predefined macro to link to the latest version of the documentation for most libraries. There are a few exceptions, where an absolute link is required, noted in the following table. To use a macro in your documentation, cut and paste the *Macro* entry from the table. The *Link* entry shows how the macro will work, and how it will appear in your documentation, once published.
+
+If you are _not_ using the AsciiDoc/Antora system, you can use the links below to retrieve the latest URL for any library.
```

Comment:
These should be the links for all libraries. Even when it supports the AsciiDoc/Antora system.